test/utils/get_localize_status_test.dart

```diff
@@ -46,12 +46,12 @@ void main() async {
 
       expect(textWidget.data, isNotNull);
 
-      expect(textWidget.data, equals('Active now'));
+      expect(textWidget.data, equals('Online'));
```

Comment:
How about store expect value in a constant

Reply:
done